### PR TITLE
Translated the Code of conduct to Japanese

### PR DESCRIPTION
**Pull request by aliya-rahmani**
References #[Issue number] 
*Please link to the related issue, using the [references] keyword only if the PR completes the entire issue*

**Proposed Changes**

*Description of changes in this PR*

**Testing Plan**

*How do the reviewers QA this? Feel free to write unit tests or describe manual testing steps here*

### CODE REVIEW

**Comment by tuttiq** [comment]
Thank you so much for your PR @aliya-rahmani ! Can you copy the translation on a separate file so we don't delete the English version? Maybe a new file like `CODE_OF_CONDUCT-JA.md` ? 

Also can you add some simple information to your PR description? 🙂 

Thank you!

**Comment by ann-kilzer** [comment]
This is great. @tuttiq, we should also add a disclaimer to both versions that in the case of any ambiguity in the Code of Conduct, the English version takes priority. Sound good?

**Comment by aliya-rahmani** [comment]
If you can add the disclaimer, I'll edit the new edited file.

**Comment by tuttiq** [comment]
Since you opened a new PR for the separate file, I'll close this one! 🙏 
https://github.com/WWCodeTokyo/speak-her-db/pull/165

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,93 +1,71 @@
...
-
-_Revision 2.3. Posted 6 March 2017._
-
-_Revision 2.2. Posted 4 February 2016._
-
-_Revision 2.1. Posted 23 June 2014._
-
-_Revision 2.0, adopted by the [Stumptown Syndicate](http://stumptownsyndicate.org) board on 10 January 2013. Posted 17 March 2013._
-
-## 11. Addendum
```
Comment (ann-kilzer) [inline CODE_OF_CONDUCT.md:89]: I notice part 11 is missing from the translation.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,93 +1,71 @@
...
- * Advocating for, or encouraging, any of the above behavior.
+8.範囲
+すべてのコミュニティ参加者（寄稿者、有料またはその他、スポンサー、およびその他のゲスト）は、オンラインおよび対面のすべてのコミュニティ会場、および関連するすべての1対1のコミュニケーションにおいて、この行動規範を遵守することを期待しています。コミュニティビジネスへ。
 
-## 5. Consequences of Unacceptable Behavior
+この行動規範とそれに関連する手順は、コミュニティ活動の範囲外で発生する容認できない行動にも適用されます。そのような行動は、コミュニティメンバーの安全と福祉に悪影響を与える可能性があります。
 
-Unacceptable behavior from any community member, including sponsors and those with decision-making authority, will not be tolerated.
+9.連絡先情報
+talkherjp@gmail.com
```
Comment (ann-kilzer) [inline CODE_OF_CONDUCT.md:58]: speakherjp@gmail.com

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -1,93 +1,71 @@
-# Citizen Code of Conduct
+市民行動規範
+1。目的
```
Comment (ann-kilzer) [inline CODE_OF_CONDUCT.md:2]: Let's be sure to use proper [Markdown syntax](https://www.markdownguide.org/) and headers with the `#` notation. Are you new to Markdown? Can we help?

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -1,93 +1,71 @@
...
-_Revision 2.1. Posted 23 June 2014._
-
-_Revision 2.0, adopted by the [Stumptown Syndicate](http://stumptownsyndicate.org) board on 10 January 2013. Posted 17 March 2013._
-
-## 11. Addendum
-
-Any in-person events organized via Women Who Code, including events related to the SpeakHer project, are covered by the Women Who Code [Code of Conduct](https://www.womenwhocode.com/codeofconduct).
-
-Third-party events organized by users of the [SpeakHer Website](https://speakher.jp) may have their own community rules. In the scope of a Speaking Event where a speaker has been selected from the Speak Her DB, We request that the speaker is treated following the expectations in this Code of Conduct.
+スタンプタウン-シンジケートボードによって採用されたリビジョン2.0
```
Comment (ann-kilzer) [inline CODE_OF_CONDUCT.md:71]: We should retain the urls to the original source as well.

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -1,93 +1,71 @@
...
-The following behaviors are expected and requested of all community members:
+コミュニティメンバーが容認できない行動をとった場合、コミュニティオーガナイザーは、警告なしに（有料イベントの場合は返金なしで）コミュニティからの一時的な禁止または永久的な追放を含め、適切と思われる行動をとることができます。
 
- * Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
- * Exercise consideration and respect in your speech and actions.
- * Attempt collaboration before conflict.
- * Refrain from demeaning, discriminatory, or harassing behavior and speech.
- * Be mindful of your surroundings and of your fellow participants. Alert community leaders if you notice a dangerous situation, someone in distress, or violations of this Code of Conduct, even if they seem inconsequential.
+6.報告ガイドライン
+容認できない行動を起こしたり目撃したりする場合、またはその他の懸念がある場合は、できるだけ早くコミュニティ主催者に通知してください。 talkherjp@gmail.com。
```
Comment (ann-kilzer) [inline CODE_OF_CONDUCT.md:45]: speakherjp@gmail.com

I'm wondering why the email address has been changed.

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -1,93 +1,71 @@
...
 
-## 1. Purpose
+この行動規範は、私たちのコミュニティに参加するすべての人々に対する私たちの期待と、容認できない行動の結果を概説しています。
 
-A primary goal of Speak Her DB is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+Speak Her DBに参加するすべての人を招待して、すべての人に安全で前向きな体験を提供できるように支援します。
 
-This code of conduct outlines our expectations for all those who participate in our community, as well as the consequences for unacceptable behavior.
+2. [ソース/文化/技術]市民権を開く
+この行動規範の補足的な目標は、参加者に私たちの行動とコミュニティへの影響との関係を認識して強化することを奨励することにより、オープン[ソース/文化/技術]の市民権を高めることです。
```
Comment (ann-kilzer) [inline CODE_OF_CONDUCT.md:10]: So this appears to be a verbatim Google Translate conversion. 

"A supplemental goal of this Code of Conduct is to increase open [source/culture/tech] citizenship by encouraging participants to recognize and strengthen the relationships between our actions and their effects on our community." 

=>

この行動規範の補足的な目標は、参加者に私たちの行動とコミュニティへの影響との関係を認識して強化することを奨励することにより、オープン[ソース/文化/技術]市民権を高めることです。

Given this document lays out community behavior expectations, we'll actually need a human translation.

**Review by ann-kilzer — COMMENTED** [review]
Thanks for your contribution. Is this a Google Translate conversion? We will probably need a human translation for this document. See my comment below.

**Pull request closed without merging**